features/step_definitions/et3_respondents_details_page_steps.rb

```diff
@@ -0,0 +1,34 @@
+Given(/^I am on the ET3 respondents details page$/) do
+  @respondent = FactoryBot.create(:et3_respondent)
+  start_page.load
+  start_page.next
+  expect(respondents_details_page).to be_displayed
+end
+
+When(/^I successfully submit all the respondents details$/) do
+  et3_answer_respondents_details
+end
+
+When(/^I successfully submit the required respondents details only$/) do
+  et3_answer_required_espondents_details
+end
+
+Then(/^I should be taken to the claimants details page$/) do
+  expect(claimants_details_page).to be_displayed
+end
+
+When(/^I click on next without providing the required respondents details$/) do
+  respondents_details_page.next
+end
+
+Then(/^I should see the error message saying the respondents details cant be blank$/) do
+  expect(respondents_details_page).to have_error_header
+  # functionality is wrong for case number
```

Comment:
removed unused code please?

Reply:
As discussed, changes are needed, these relate to https://tools.hmcts.net/jira/browse/RST-1127